apis/r/tests/testthat/test-shape.R

```diff
@@ -1,107 +1,222 @@
 test_that("SOMADataFrame shape", {
-  #uri <- withr::local_tempdir("soma-dataframe-shape")
-  uri <- "/tmp/fooze"
   asch <- create_arrow_schema()
 
   index_column_name_choices = list(
     "soma_joinid",
     c("soma_joinid", "int_column"),
     c("soma_joinid", "string_column"),
+    c("string_column", "int_column"),
```

Comment:
Can we add a comment indicating the intentional duplication here?
```suggestion
    c("string_column", "int_column"), # duplicate intentional to match `domain_at_create_choices`
```